conans/client/generators/__init__.py

```diff
@@ -223,7 +223,8 @@ def write_toolchain(conanfile, path, output):
                 conanfile.generate()
 
     # tools.env.virtualenv:auto_use will be always True in Conan 2.0
-    if conanfile.conf["tools.env.virtualenv:auto_use"] and conanfile.virtualenv:
+    auto_use = conanfile.conf["tools.env.virtualenv:auto_use"]
```

Comment:
This will activate the new VirtualEnv, which might be breaking, when the user doesn't define anything at all at [conf].

Reply:
🤔 then who is activating it when I use the CMakeDeps + tool chain? Not very intuitive

Reply:
Current tests, either use explicitly ``generators = "VirtualEnv"``, or set up the ``auto_use`` conf. But it should be either opt-in, to not break.